deployable/java/src/main/java/envtest/deployable/Snippets.java

```diff
@@ -31,13 +32,35 @@ public void simplelog(Map<String,String> args){
         // pull out arguments
         String logText = args.getOrDefault("log_text", "simplelog");
         String logName = args.getOrDefault("log_name", "test");
-        String severityString = args.getOrDefault("severity", "DEFAULT");
+        String severityString = args.getOrDefault("severity", "CRITICAL");
```

Comment:
why is default severity "CRITICAL"?

Reply:
I usually set it to a high severity if unspecified to make sure logs stand out. I'm fine with changing it though